src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,41 @@
+REM 
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
```

Comment:
I think the lab scripts may come from a different repo or be in a private location on the machine, so I don't think %~dp0 will work.  However, this script should definitely check that %1 has been set and points to a folder that contains what it expects (with an appropriate error if it does not).